### PR TITLE
Add migration

### PR DESCRIPTION
**Pull request by ciraganenicole**
In this pull request I:

- Create migration by adding relationships between tables

### CODE REVIEW

**Review by AyoubMs — COMMENTED** [review]
Your migrations are complete.
It's time to merge.
Congratulations

File: db/migrate/20220726073904_add_user_ref_to__food.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToFood < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :foods, :user, null: false, foreign_key: true
+  end
+end
```
Comment (Achilles-Dev) [inline db/migrate/20220726073904_add_user_ref_to__food.rb:7]: Food does not have a relationship with a user. I suggest you remove the `user_id` from foods table

File: db/migrate/20220726073301_create_recipes.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+class CreateRecipes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :recipes do |t|
+      t.string  :name
+      t.integer :preparation_time, default: 0
+      t.integer :cooking_time, default: 0
```
Comment (Achilles-Dev) [inline db/migrate/20220726073301_create_recipes.rb:8]: The data type for `preparation_time` and `cooking_time` should be time and not `integer`.

**Review by Achilles-Dev — COMMENTED** [review]
Hi, @ciraganenicole 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

**Pull request merged**